model/src/service/path.py

```diff
@@ -30,5 +28,4 @@ def process_configfile():
 
 config_data = process_configfile()
 
-HOP = config_data["Hop"]
-# TYPE = config_data["Type"]
\ No newline at end of file
+SERVICE_CONFIG = config_data[0] #["endpoints"][0]
```

Comment:
As PR #31 will change the configmap data to a complete json, I would suggest to fix it before merging it.

Reply:
Yes, updating code according to new json format...